app/(gcforms)/[locale]/(user authentication)/auth/register/action.ts

```diff
@@ -131,9 +133,11 @@ export const register = async (
 
   try {
     await cognitoIdentityProviderClient.send(signUpCommand);
+    logMessage.info("HealthCheck: cognito sign-up success");
   } catch (err) {
-    // if there is an error, forward the status code and the error message as the body
+    logMessage.info("HealthCheck: cognito sign-up failure");
```

Comment:
For all of the error state logs, what would you think of changing them to use `error` instead of `info`?  That would give us the specific healthcheck messages and a generic `ERROR` search term if we wanted to group all failure conditions.
```suggestion
    logMessage.error("HealthCheck: cognito sign-up failure");
```

Reply:
The reason is that they would also appear in our Slack notifications..